Python/Product/Django/Django.csproj

```diff
@@ -278,7 +278,7 @@
           <HintPath>$(SolutionDir)\References\Dev15.0\Microsoft.Web.Core.dll</HintPath>
         </Reference>
         <Reference Include="Microsoft.Web.Editor, Version=$(VSTarget).0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
-          <HintPath>$(DevEnvDir)\Extensions\Microsoft\Web Tools\Editors\Microsoft.Web.Editor.dll</HintPath>
+          <HintPath>$(SolutionDir)\References\Dev15.0\Microsoft.Web.Editor.dll</HintPath>
```

Comment:
You don't need a hint path - we automatically search `References\Dev$(VsTarget)`